src/charactersheet/viewmodels/dm/encounter_sections/npc_section/index.html

```diff
@@ -41,17 +41,18 @@
           href="#" class="hidden-md hidden-lg col-md-5"></td>
       <td data-bind="text: longDescription, click: $parent.editNPC"
           href="#" class="hidden-sm hidden-xs col-md-5"></td>
-      <!-- ko if: $parent.shouldShowPushButton -->
-      <td class="text-right col-xs-1">
-        <a data-bind="click: $parent.pushModalToPlayerButtonWasPressed" href="#">
-          <i class="fa fa-paper-plane-o fa-color-hover"></i>
+      <!-- ko if: $parent.shouldShowExhibitButton -->
+      <td class="text-right col-md-1">
+        <a data-bind="click: $parent.toggleExhibit" href="#" title="Send Player's Exhibit">
+          <i data-bind="css: { exhibitActiveIcon: isExhibited }" class="fa fa-desktop fa-color-hover">
+          </i>
         </a>
       </td>
       <!-- /ko -->
-      <!-- ko ifnot: $parent.shouldShowPushButton -->
-      <td class="text-right col-xs-1">
-        <span class="fa fa-paper-plane-o fa-disabled" style="cursor:pointer;"
-          data-bind="popover: { content: 'Join a Party to send image to Player\'s Chat' }">
+      <!-- ko ifnot: $parent.shouldShowExhibitButton -->
+      <td class="text-right col-md-1">
+        <span class="fa fa-desktop fa-disabled" style="cursor:pointer;"
+          data-bind="popover: { content: 'Join a Party send Player\'s Exhibit' }">
```

Comment:
This should still probably says `Join a Party to send image to Player\'s Exhibit` or something like it.